features/git-kill/current_branch/on_feature_branch/without_remote_origin/with_open_changes.feature

```diff
@@ -0,0 +1,50 @@
+Feature: git kill: killing the current feature branch without a tracking branch (with open changes and without remote repo)
+
+  (see ../without_tracking_branch/with_open_changes.feature)
+
+
+  Background:
+    Given I have a feature branch named "feature"
+    And my repo does not have a remote origin
+    And I have a local feature branch named "dead-feature"
+    And the following commits exist in my repository
+      | BRANCH       | LOCATION | MESSAGE         | FILE NAME        |
+      | feature      | local    | good commit     | good_file        |
```

Comment:
In https://github.com/Originate/git-town/pull/451 I am improving the naming of the branches and files. Please adopt this style here.

```cucumber
And the following commits exist in my repository
  | BRANCH          | LOCATION | MESSAGE                | FILE NAME            |
  | other-feature   | local    | other feature commit   | other_feature_file   |
  | current-feature | local    | current feature commit | current_feature_file |
And I am on the "current-feature" branch